wikipedia-dark.user.css

```diff
@@ -283,6 +345,32 @@
     background-color: transparent !important;
     border-color: #555 !important;
   }
+  .mw-editfont-monospace {
+    background-color: #222;
+    color: #9a9a9a;
+  }
+  .userSummaryButtons a, #userSummaryButtons a {
+    background-color: #333;
+    border-color: #333;
+  }
+  .userSummaryButtons a:hover, #userSummaryButtons a:hover {
+    background-color: #444;
+    border-color: #444;
+  }
+  #btn_opisy_zm_waxon_off {
+    background-color: #333 !important;
+    border-color: #333 !important;
+  }
```

Comment:
Same with this button, the color scheme should be along 

```css
    background-color: #261d08 !important;
    border-color: #542 !important;
```

Reply:
Implemented